checkout/index.md

```diff
@@ -68,14 +70,14 @@ sequenceDiagram
 
             Payer ->>+ Merchant: Start Checkin
                 Merchant ->>+ SwedbankPay: POST /psp/consumers
-                    SwedbankPay -->>- Merchant: rel:view-consumer-identification [1]
+                    SwedbankPay -->>- Merchant: rel:view-consumer-identification ❶
```

Comment:
These are the hardest to read symbols I've come across in quite a while. Is there an alterantive to these? When rendered in a browser they are quite difficult to read in the diagram.

Reply:
Hm, I think they look just fine here. Not here in GitHub, but in the actual rendered web page.

![image](https://user-images.githubusercontent.com/12283/68833334-821be400-06b3-11ea-967d-ca2986bb0883.png)

Reply:
I find them really difficult to read. Might be my screen.
![hjntyrhjdxjnyr](https://user-images.githubusercontent.com/42067409/68833507-e9d22f00-06b3-11ea-8aaa-c132dc1d209e.PNG)

Reply:
Retina resolution helps, yes. I've replaced the characters now. Are the new ones better?

Reply:
I'll ask my squad leader to upgrade my screens.
The new ones are easier to read yes. Thank you.

Reply:
![Retina All The Things](https://user-images.githubusercontent.com/12283/68835514-8d254300-06b8-11ea-938c-521c6c1c5a8f.png)